tardis/montecarlo/formal_integral.py

```diff
@@ -124,4 +128,9 @@ def make_source_function(self):
         att_S_ul =  ( wave * (q_ul * e_dot_u) * t  / (4*np.pi) )
 
         result = pd.DataFrame(att_S_ul.as_matrix(), index=transitions.transition_line_id.values)
-        return result.ix[atomic_data.lines.index.values].as_matrix(), e_dot_u
+        att_S_ul = result.ix[atomic_data.lines.index.values].as_matrix()
```

Comment:
if you just do `.values` that is maybe clearer that you just want the array and not as a real matrix.